.gitignore

```diff
@@ -34,3 +34,5 @@ build/
 
 # unless supporting rvm < 1.11.0 or doing something fancy, ignore this:
 .rvmrc
+
+node_modules
```

Comment:
??? You should NOT have a folder like that in your project. ++ for adding it to `.gitignore`, but please review your folder structure.